commit 5dbae5f7a91bb1e3d89834451cc4b050f7f449b4
Author: Joe Walker <jwalker@mozilla.com>
Date:   Wed Nov 10 12:56:39 2010 +0000

    Gutter should be GutterLayer.
    Unsure how this got to be in this state.

diff --git a/plugins/ace/lib/ace/virtual_renderer.js b/plugins/ace/lib/ace/virtual_renderer.js
index 33f6f62..8f3544a 100644
--- a/plugins/ace/lib/ace/virtual_renderer.js
+++ b/plugins/ace/lib/ace/virtual_renderer.js
@@ -41,7 +41,7 @@ var oop = require("ace/lib/oop");
 var lang = require("ace/lib/lang");
 var dom = require("ace/lib/dom");
 var event = require("ace/lib/event");
-var Gutter = require("ace/layer/gutter");
+var GutterLayer = require("ace/layer/gutter");
 var MarkerLayer = require("ace/layer/marker");
 var TextLayer = require("ace/layer/text");
 var CursorLayer = require("ace/layer/cursor");
